pkg/artifactory/resource_artifactory_local_cargo_repository.go

```diff
@@ -0,0 +1,42 @@
+package artifactory
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceArtifactoryLocalCargoRepository() *schema.Resource {
+	const packageType = "cargo"
+
+	var cargoLocalSchema = mergeSchema(baseLocalRepoSchema, map[string]*schema.Schema{
+		"cargo_anonymous_access": {
```

Comment:
Do we have to call it `cargo_anonymous_access` when it's already in the cargo resource?

Reply:
changed to anonymous_access